docs/source/installation.rst

```diff
@@ -8,20 +8,20 @@ Download and extract precompiled binary for your OS and platform from `the relea
 
 Alternatively, you can compile from source code (Golang 1.20+ required)
 
-.. code::
+.. code-block:: bash
 
     make receptor
 
 Test the installation with
 
-.. code::
+.. code-block:: bash
 
     receptor --help
     receptor --version
 
 The preferred way to interact with receptor nodes is to use the receptorctl command line tool
```

Comment:
```suggestion
The preferred way to interact with receptor nodes is to use the ``receptorctl`` command line tool
```

Reply:
Should "receptor" start with a capital R since it is the name?

Reply:
added both suggestions